src/ctap/main_hid.rs

```diff
@@ -30,6 +30,12 @@ pub struct MainHid {
     wink_permission: TimedPermission,
 }
 
+impl Default for MainHid {
```

Comment:
Where is this used?

Reply:
In here, nowhere. This was added (as countless other times) to satisfy the clippy lints.

Reply:
We can move the implementation of `new` to `default` then. And call `::default()` instead of `::new()`.